HotSpot/View Controllers/ParkingSearchViewController.m

```diff
@@ -32,8 +33,15 @@ - (void)viewDidLoad {
     self.searchTableView.delegate = self;
     self.searchTableView.dataSource = self;
     
+    
+    self.locationManager.delegate = self;
+    self.locationManager =[[CLLocationManager alloc]init];
+    self.locationManager.desiredAccuracy = kCLLocationAccuracyBest;
+    [self.locationManager requestWhenInUseAuthorization];
+    [self.locationManager startUpdatingLocation];
```

Comment:
looks like you need location in several new controllers, and you're creating a new location manager for each one. Can you have a HotSpot location manager class that handles this? it should have startLocationUpdates that starts updates if it hasn't already, stopLocationUpdates that stops updating location when location is not needed, and getLatestLocation to return the latest location. the location manager would be a singleton